src/main/java/uk/gov/pay/connector/charge/service/ChargeService.java

```diff
@@ -430,8 +430,8 @@ public ChargeEntity transitionChargeState(
                 .ifPresent(eventType -> {
                     if (shouldEmitPaymentStateTransitionEvents) {
                         PaymentStateTransition transition = new PaymentStateTransition(chargeEventEntity.getId(), eventType);
-                        paymentStateTransitionQueue.offer(transition);
-                        logger.info("Offered payment state transition to emitter queue [from={}] [to={}] [chargeEventId={}] [chargeId={}]", fromChargeState, targetChargeState, chargeEventEntity.getId(), charge.getExternalId());
+                        stateTransitionQueue.offer(transition);
+                        logger.info("Offered payment state transition to emitter queue [from={}] [to={}] [chargeId={}]", fromChargeState, targetChargeState, chargeEventEntity.getId());
```

Comment:
This ID is not the `chargeId`

```suggestion
                        logger.info("Offered state transition to emitter queue [from={}] [to={}] [chargeEventId={}] [chargeId={}]", fromChargeState, targetChargeState, chargeEventEntity.getId(), charge.getExternalId());
```

Reply:
:+1